.github/workflows/quality.yaml

```diff
@@ -5,6 +5,22 @@ on:
       - main
 
 jobs:
+  markdown-link-check:
+    name: Broken markdown links
+    runs-on: ubuntu-latest
+    steps:
+      - name: Code checkout
+        uses: actions/checkout@v2
+
+      - name: Run link check
+        uses: gaurav-nelson/github-action-markdown-link-check@v1
+        with:
+          use-quiet-mode: 'no'
+          use-verbose-mode: 'yes'
+          check-modified-files-only: 'yes'
```

Comment:
This won't catch existing links breaking due to path renames. It wouldn't have found the issue I recently fixed until someone edited that file.
Have you added it because of the issues (false positives) it finds about the github relative urls (`../../issues/...`)? In that case I'd consider replacing them with absolute URLS (unless we can find a better way to whitelist that)

Reply:
Ps. I see at https://github.com/netdata/netdata/blob/master/.mlc_config.json we can add whitelists